src/disco/containers/InstallButton.js

```diff
@@ -29,12 +32,55 @@ export class InstallButton extends React.Component {
     downloadProgress: 0,
   }
 
+  componentDidMount() {
+    const { dispatch, guid, installURL, slug } = this.props;
+    this.addonManager = new AddonManager(guid, installURL, this.statusChanged);
+    this.addonManager
+      .getAddon()
```

Comment:
The call `this.addonManager.getAddon()` returns the promise. So it follows it looks nicer to have:

```javascript
// Call the thing that returns the promise
this.addonManager.getAddon()
  .then(() => {
     // Make the then stand out as this is handling the successful promise resolution.
  })
  .catch(() => {
     // Catch as needed.
  });
```

Make the flow really clear. Indenting just the method as well makes the flow layout less clear.